/minimal_subscriber:
  ros__parameters:
    laser_scan:
      topic_name: "/scan"
      queue_size: 10